OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -296,11 +296,15 @@ public interface IFacingInfo : ITraitInfoInterface { int GetInitialFacing(); }
 
 	public interface ICrushable
```

Comment:
You've already made all the implementations explicit, so can you please add `[RequireExplicitImplementation]` so they stay that way?